### DISCUSSION TITLE
code intellisense can't display all

### DISCUSSION THREAD

**Discussion by ManHand1996** [microsoft/pylance-release#3459] [Enhancement]
for example: when I dot input 'ref' to find 'getrefcount' method  in 'sys' package.
I hope pylance work like Pycharm which displays all methods which include the input words
Did I need some setting to achieve?

Has anyone else had this problem?

In vscode:
![vscode_pylance_intellisence](https://user-images.githubusercontent.com/31920754/194702694-aa3aff11-2156-4cce-9abb-48db1cd71745.png)

My vscode only starts with input words that matched

![vscode2](https://user-images.githubusercontent.com/31920754/194702967-dc7dcaa1-74c3-4bcf-8567-5083a540f79c.png)

In Pycharm:
![pycharm_intelience](https://user-images.githubusercontent.com/31920754/194702697-5961f072-c4a9-41ca-913e-13179a5f741a.png)

Please told me how to do that. Many Thanks.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
